java-checks-test-sources/default/src/main/java/checks/VisibleForTestingProtectedUsageCheck/IssueMyObject.java

```diff
@@ -0,0 +1,14 @@
+package checks.VisibleForTestingProtectedUsageCheck;
+
+public class IssueMyObject {
+
+  IssueMyObject() {
+  }
+
+  // androidx.annotation.VisibleForTesting
+  // @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)
+  // where VisibleForTesting.PROTECTED = 4
+  @VisibleForTesting(otherwise = 3, othertestcase=1)
```

Comment:
Do we have a test:
`@VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)`

Reply:
I meant a test case like this in`IssueMyObject `:
`  @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED, othertestcase=1)
  String bar; `

Inside VisibleForTesting, we need to add an `int PROTECTED = 4; `

And in an `IssueService` something like this:
`  public String test2() {
    return new IssueMyObject().bar; // Noncompliant
  } `